benchmarks/infrastructure/gcp/terraform/README.md

```diff
@@ -0,0 +1,46 @@
+# Create infrastructure with Terraform
+
+1. Install [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/gcp-get-started).
+
+2. Create and download [service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-console).
+   The path to the file is required in the next step (`credentials_file`).
+
```

Comment:
I would add:
Make sure the service account has the [permissions](https://console.cloud.google.com/iam-admin/iam) listed in prerequisites.

Reply:
done